cmd/root.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright © 2022 Nethermind hello.nethermind.io
+
+*/
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var cfgFile string
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "1Click",
+	Short: "A brief description of your application",
+	Long: `A longer description that spans multiple lines and likely contains
```

Comment:
Description miss match.. Also we don't need the cmd/cli.go if you have the cmd/root.go

Reply:
Need to update the description, I'm not finished with this cmd yet. 
The thought process around this commands is:

```
> 1click
# Open the UI
> 1click cli --execution ... --consensus ...
# Run quick setup without UI. Proper flow for bots too
```

Reply:
Please read the PR description for what I intend to do in this PR